docs/DafnyRef/Types.md

```diff
@@ -362,6 +362,9 @@ unsigned arithmetic modulo 2^{number of bits}, like 2's-complement machine arith
 -----------------|------------------------------------
   `-`            | bit-limited negation (unary minus)
   `!`            | bit-wise complement
+-----------------|------------------------------------
+  .RotateLeft(n) | rotates bits left (up) by n bit positions
```

Comment:
(up) and (down) make little sense to me. Why do you need them in parentheses ?

Reply:
I just thought that left and right are presuming a bit-ordering, but I'll remove the up and down and add some examples instead.